content/en/docs/concepts/policy/limit-range.md

```diff
@@ -0,0 +1,390 @@
+---
+reviewers:
+- sftim
+title: LimitRange
+content_template: templates/concept
+weight: 10
+---
+
+{{% capture overview %}}
+
+By default, containers run with unbounded [compute resources](/docs/user-guide/compute-resources) on kubernetes cluster.
+With Resource quotas, cluster administrators can restrict the resource consumption and creation on a namespace basis.
+Within a namespace, a Pod/Container can still consume as much CPU and memory as the defined namespace's resource quota.

```

Comment:
Hi @nelvadas. Great work. Some small suggestions:
- should the title be two words?
- line 11, on a Kubernetes cluster or on Kubernetes clusters
- line 13, a Pod or Container
- line 14, There is ... Pod or Container could monopolize ...

Reply:
Fixed, the task section is already indexed at the end for next steps.
/tasks/administer-cluster/manage-resources/cpu-constraint-namespace/#create-a-limitrange-and-a-pod but do not really intend to discuss LimitRange in details.